### ISSUE TITLE
Should not error out "use --update to update it" if we pass `--update` flag

### ISSUE THREAD

**Issue by Dentrax** [tldr-pages/tldr-c-client#62]
Since we are passing the `--update` flag, I think throwing the following error might be unnecessary. We can write different logs here to improve the error message: [^1]

```bash
$ tldr --update
Local data is older than two weeks, use --update to update it.
```

```bash
$ tldr --version
tldr v1.3.0 (v1.3.0)
```

[^1]: https://github.com/tldr-pages/tldr/issues/7376

**Comment by marchersimon** [comment]
@Dentrax this is really weird, because I don't have this issue. I don't even know how this issue can possibly arise, because this message get's printed here:
https://github.com/tldr-pages/tldr-c-client/blob/af7501fd11a7045d0bf6282e8b333148347de482/src/local.c#L59

and the function is called here:
https://github.com/tldr-pages/tldr-c-client/blob/af7501fd11a7045d0bf6282e8b333148347de482/src/tldr.c#L117

Full context:
```C
if (!update_flag) {
    check_localdate();
}
```

So, when `update_flag` is `true`, the function can't get called. Where did you install the client and could you maybe recompile it yourself to see if the issue still exists?

**Comment by MasterOdin** [comment]
This was fixed in 1cf890db5fdd63e08dd364dd02e072c0639862d4 as part of the 1.4.0 release
